src/Microsoft.ML.TensorFlow/TensorflowTransform.cs

```diff
@@ -1077,15 +1080,19 @@ public TFTensor GetBufferedBatchTensor()
         }
     }
 
+    /// <summary>
+    /// Estimator for the <see cref="TensorFlowTransformer"/>.
+    /// </summary>
     public sealed class TensorFlowEstimator : IEstimator<TensorFlowTransformer>
     {
         private readonly IHost _host;
-        private readonly TensorFlowTransformer.Options _args;
+        private readonly TensorFlowTransformer.Options _options;
         private readonly TensorFlowModelInfo _tensorFlowModel;
         private readonly TFDataType[] _tfInputTypes;
         private readonly ColumnType[] _outputTypes;
         private TensorFlowTransformer _transformer;
 
+        [BestFriend]
```

Comment:
>[BestFriend] [](start = 8, length = 12)

I fount this suspicious. How come you care only about one constructor been best friend while we have 4 of them?

Reply:
It needs to be BestFriend for the DNN assemblies, and they only use this constructor.

---
In reply to: [254453417](https://github.com/dotnet/machinelearning/pull/2367#discussion_r254453417) [](ancestors = 254453417)